charts/rasa-x/templates/rasa-deployments.yaml

```
@@ -84,6 +84,8 @@ spec:
         - name: "RASA_TELEMETRY_ENABLED"
           value: "false"
         {{ end }}
+        - name: "MPLCONFIGDIR"
+          value: "/tmp/"
```

I think we should use a subdirectory like 
```suggestion
          value: "/tmp/.matplotlib"
```